CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # CHANGELOG
 
+## `v10.2.0-beta`
+### ARM
+
+| api                                 | version            | note                                |
+|:------------------------------------|:-------------------|:------------------------------------|
+| arm/cosmos-db                       | 2015-04-08         | new                                 |
+| arm/mysql                           | 2017-04-30-preview | new                                 |
+| arm/postgresql                      | 2017-04-30-preview | new                                 |
```

Comment:
👍 awesome!

Out of interest is the old DocumentDB SDK being deprecated in v11 in favour of the renamed CosmosDB SDK?